### PR TITLE
Fabric Accessibility - High contrast mode

### PR DESCRIPTION
**Pull request by philworthington**
High contrast mode pass with changes to the following:
- New Fabric colors for high contrast
- People tokens in people picker
- Callout beak
- Radio buttons and choice fields
- Contextual menu hover, focus, selected states
- Dialog border, overlay opacity
- Dropdowns hover and focus
- Disabled dropdowns
- Pivot selection
- Pivot redesign - new indicator and state colors
- Spinner visibility
- Label disabled color
- Toggle clean up 
- Progress bar visibility
- Textfield state colors
- Fix for underline textfield active state in all display modes

Set Windows into 'High Contrast Black' or 'High Contrast White' and open Edge/IE to review changes.

### CODE REVIEW

File: src/components/Pivot/Pivot.scss [hunk 1]
```diff
@@ -9,7 +9,7 @@
 .ms-Pivot {
   @include ms-font-m;
   @include ms-u-normalize;
-  height: 30px;
+  height: 40px;
```
Comment (mikewheaton) [inline src/components/Pivot/Pivot.scss]: Was this a bug in Pivot where it should have been 40px?

File: src/sass/_Fabric.Color.Variables.scss [hunk 2]
```diff
@@ -77,3 +77,13 @@ $ms-color-alert:             $ms-color-orange;
 $ms-color-alertBackground:   $ms-color-neutralLighter;
 $ms-color-error:             $ms-color-redDark;
 $ms-color-errorBackground:   #fde7e9;
+
+
+//== High contrast colors
+//
+$ms-color-contrastBlackDisabled: #00ff00;
```
Comment (mikewheaton) [inline src/sass/_Fabric.Color.Variables.scss]: We'd usually document new colors, but I don't think we'd provide classes for these (e.g. `.ms-bgColor-contrastBlackDisabled`) for general use.

**Comment by notandrew** [comment]
It looks good to me -- we should fix Link as well.  I'll file a bug on that so we don't hold up this change.

**Comment by philworthington** [comment]
Yeah there wasn't a bug out there for Link so it wasn't looked at but this is a great catch. I've updated Link with the correct colors and it should be fixed now.

File: src/components/Pivot/Pivot.scss (hunk 1 above)
Comment (philworthington) [inline src/components/Pivot/Pivot.scss]: There are new redlines for Pivot that were implemented here along with the high contrast pass. 40px height are a part of the new design along with the indicator at the bottom and an update to the colors being used.

**Comment by battletoilet** [comment]
Anchor links are invisible in HC #1
hc-anchorlink

**Comment by battletoilet** [comment]
HC Persona doesnt really have a specific color.
hcpersona

**Comment by battletoilet** [comment]
It looks like bottom field is kinda funky.
hc-textfield

File: src/components/ChoiceField/ChoiceField.scss [hunk 3]
```diff
@@ -37,6 +37,36 @@
 		.ms-Label {
 			color: $ms-color-neutralTertiary;
 		}
+
+		@media screen and (-ms-high-contrast: active) {
+			&:before {
+				background-color: $ms-color-contrastBlackDisabled;
```
Comment (battletoilet) [inline src/components/ChoiceField/ChoiceField.scss]: Tabs?

File: src/components/ChoiceField/ChoiceField.scss [hunk 4]
```diff
@@ -106,6 +136,16 @@
 			width: 11px;
 			height: 11px;
 			box-sizing: border-box;
+
+			@media screen and (-ms-high-contrast: active) {
+				border-color: $ms-color-white;
```
Comment (battletoilet) [inline src/components/ChoiceField/ChoiceField.scss]: The Tabbing is weird here.

File: src/components/Persona/Persona.scss [hunk 5]
```diff
@@ -219,6 +235,15 @@ $ms-color-initials-darkRed: 		#b91d47;
 		top: 10px;
 		left: 0;
 		border: 0;
+
+		@media screen and (-ms-high-contrast: active) {
+			top: 9px;
```
Comment (battletoilet) [inline src/components/Persona/Persona.scss]: Tabbing is weird, looks like a mix between tabs and spaces.

**Comment by philworthington** [comment]
Updates made to the high contrast link color so it's visible in high contrast #1. Fixes for textfield made for all display modes and states. Translate appropriate files (choicefield.scss, persona.scss) from tabs to spaces.

**Pull request merged**